fastlane/lib/fastlane/helper/adb_helper.rb

```diff
@@ -30,7 +30,7 @@ def trigger(command: nil, serial: nil)
         Action.sh(command)
       end
 
-      def device_avalaible?(serial)
+      def device_available?(serial)
```

Comment:
@joshdholtz Is this OK?

```suggestion
      def device_avalaible?(serial)
        load_all_devices
        return devices.map(&:serial).include?(serial)
      end

      def device_available?(serial)
```

Reply:
Nah, you can really use what @joshdholtz suggested at https://github.com/fastlane/fastlane/pull/15061/files#r305810440